portfolio/src/main/webapp/index.html

```diff
@@ -10,10 +10,54 @@
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+
+      <p>------------------------------</p>
+      <p>   This will hopefully turn out alright (Let's try tabbing to see how it displays in the webpage)</p>
+      <p>Update, the blank space made by tab in the code didn't show in the webpage</p>
```

Comment:
Per my previous comment: control spacing with CSS.

HTML tags are intended to convey *semantic* markup: the document structure of titles, lists, paragraphs etc. The visual appearance should be provided by CSS. I expect that traditional publishing inspired this: the author writes a manuscript. The editor adds markup (i.e. the tags). The book designer/typographer provides the CSS.